src/resolvers/types/ScheduledEvent.ts

```diff
@@ -58,7 +58,7 @@ export class ScheduledEvent implements Partial<ScheduledEventBase> {
   @Field({ nullable: true })
   instrument?: Instrument;
 
-  @Field(() => Int)
+  @Field(() => Int, { nullable: true })
   equipmentId: number;
```

Comment:
It is the best keep dataTypes and decorator consistant and ark equipment as optional if it is nullable
```
@Field(() =&gt; Int, { nullable: true })  
equipmentId?: number;
```